slides/lesson12.md

```diff
@@ -37,35 +37,64 @@ let stuff = ["John", 20, true, undefined];
 
 ---
 
-### arrays and objects
+### accessing elements
 
-the difference between arrays and objects is that values in arrays don't need a name.
+we can access elements in the array by number
+
+the numbering starts at `0`:
```

Comment:
Most of the sentences start with upper case. Should we follow that in order to have some uniformity?